slices/ca-certificates.yaml

```diff
@@ -6,6 +6,8 @@ essential:
 slices:
   data:
```

Comment:
Since openssl will only look at the `/etc/ssl/certs/ca-certificates.crt` path if it's symlinked from `/usr/lib/ssl/cert.pem`, I think we should make `ca-certificates_data` depend on `openssl_data` to make `ca-certificates` a bit standalone.